sites/main-site/src/content/blog/2025/modules-meta-yml.mdx

```diff
@@ -0,0 +1,162 @@
+---
+title: "Meet the new meta.yml"
+subtitle: "Changes to the structure of the meta.yml file for nf-core modules"
+headerImage: "https://unsplash.com/photos/magnifying-glass-mq8QogEBy00"
```

Comment:
```suggestion
headerImage: "https://images.unsplash.com/photo-1547057740-4b18aac8eed2"
```
alternatively https://images.unsplash.com/photo-1549964336-67d7d7d74ac2 to point more to the metadata/archive point of this file.